### DISCUSSION TITLE
pawnmorpher라는 모드가 일부분만 번역되어있는데 이어받아서 번역이 가능할까요?

### DISCUSSION THREAD

**Discussion by ghost** [RimWorldKorea/RMK#477] [질문]
항상 수고많으십니다.

궁금한점이 두가지 있습니다

현재 pawnmorpher라는 모드가 과거에 전부 번역되어있었지만 현재 모드가 많은 업데이트를 거처 모든 부분이 완벽히 번역이 되어있는 상태는 아닙니다.
이내용은 또한 https://github.com/RimWorldKorea/RMK/issues/432
에서 확인했습니다.

따라서 첫번째로 그부분을 제가 이어받아 번역을 완벽히 이어받고 싶은데 RMK\DATA\해당모드 폴더안 WIP.xlsx
파일을 번역후 최신화하여 보내드리면 번역 최신화가 되는건지 알고싶습니다.

두번째로는 현재 해당모드에 속해있는 엑셀파일은 모드의 최신버전에 맞게 최신화가 되어있는건지(해당 엑셀파일을 번역하면 모드번역상 누락되는 컨텐츠가 있는건지) 궁금합니다.

**Comment by RimWorldKorea** [comment]
**1.** [WIP.xlsx](https://github.com/RimWorldKorea/RMK/files/14257599/WIP.xlsx) ← 이 파일 새로 받아서 작업하시면 됩니다.
번역 중 인게임에서 검수가 필요할 경우 https://github.com/RimWorldKorea/RMK/discussions/478 에서 프로그램을 다운받아 `XLSX -> XML` 기능으로 변환하면 됩니다.

<img width="176" alt="화면 캡처 2024-02-13 163219" src="https://github.com/RimWorldKorea/RMK/assets/150018627/a84779f5-31fd-4ced-ade2-eee1d02988bf"><br>

**2.** 파일 자체는 최신 버전을 기준으로 생성된 것이지만
사용한 추출 프로그램이 미완성인 관계로 100% 커버될거라 장담하진 못합니다.
이 경우 누락된 노드의 정보를 알고있을 경우 시트 하단에 수동으로 기입해도 좋고
잘 모르겠으면 언제든지 다시 물어봐주세요
<br>

※ 번역 작업은 WIP.xlsx 내 첫 번째 시트인 Main에서 하면 됩니다.
